### DISCUSSION TITLE
Provide functionality to set the file path of the database file

### DISCUSSION THREAD

**Discussion by rkreisel** [warmuuh/milkman#151] [Ideas]
I generally work in a team. My current dev team is only 5 people, but I have been on much larger teams. It would be great if we could easily store the tests we configure  (i.e. the database file) in our solution's folder structure. That way they would be pushed to the repository with each PR.

I considered making modifications to the source myself, but my most recent significant Java experience was nearly 20 years ago.

**Comment by warmuuh** [comment]
Generally, milkman first tried to create a dB file at the current working directory. You could modify the starting script to start at a different place... Another solution that might work for your team is the [Git sync](https://github.com/warmuuh/milkman/blob/master/milkman-sync-git). It is hard to add it as configuration inside of milkam as these things are stored in that dB file :)

**Reply by warmuuh** [reply]
Thinking about it again, you can also use git`s sub-repo feature to include the workspace into another repo and have milkman git-sync point to that directory( you can use file-based git path for that) and have it dumped there
